templates/firefox-ios.xliff

```diff
@@ -4123,6 +4231,62 @@ Private Tab</source>
         <note>On the main menu, in the Save submenu, the title for the menu component that allows the user to add this site to the reading list.</note>
       </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.NightMode.Off.Title.v132" xml:space="preserve">
+        <source>Turn off Night Mode</source>
+        <note>On the main menu, the accessibility label for the action that will turn Night Mode off in the application.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.NightMode.On.Title.v132" xml:space="preserve">
+        <source>Turn on Night Mode</source>
+        <note>On the main menu, the accessibility label for the action that will turn Night Mode on in the application.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.NightMode.Subtitle.v132" xml:space="preserve">
+        <source>Night Mode</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Night Mode tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.Print.Subtitle.v132" xml:space="preserve">
+        <source>Print</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Report Print tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.Print.Title.v132" xml:space="preserve">
+        <source>Print</source>
+        <note>On the main menu, the accessibility label for the action that will take the user to the Print module in the application.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.Off.Title.v132" xml:space="preserve">
+        <source>Turn off Reader View</source>
+        <note>On the main menu, the accessibility label for the action that will turn the reader view on for the current website.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.On.Title.v132" xml:space="preserve">
+        <source>Turn on Reader View</source>
+        <note>On the main menu, the accessibility label for the action that will turn the reader view on for the current website.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.Subtitle.v132" xml:space="preserve">
+        <source>Reader View</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Reader View tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReportBrokenSite.Subtitle.v132" xml:space="preserve">
+        <source>Report</source>
```

Comment:
Is this a noun or a verb? What is the context around this? The comment seems more like a developer comment rather than a comment to help localizers understand the context (as all comments in this PR)